src/game/stage/rings_scatter.c

```diff
@@ -438,14 +579,26 @@ void RingsScatterSingleplayer_NormalGravity(void)
         }
 
         {
+#if (GAME == GAME_SA2)
             u16 sprFlags = ring->unk10;
             ring->unk10 &= ~0x3;
             ring->unk10 |= (sprFlags + 1) & 0x3;
+#endif
             ring->unkC--;
         }
     }
 }
 
+#define USE_HITBOX_RECT 1
```

Comment:
Do these need to be added back?

Reply:
Well, they're still used. But of course I don't want to insert something that's not needed. ^^